### DISCUSSION TITLE
How to authenticate with cypress for vercel password protection

### DISCUSSION THREAD

**Discussion by rafael-pina** [vercel/community#748] [Help] [answered]
Hey guys!

We are using cypress for the integration tests and we have set dynamic deployments with the password protection feature. When running the tests I get Unauthorized when trying to access the newly generated domain and I'm wondering if there is a way to set an access token to bypass this authentication.

**Comment by swarnava** [comment, marked as answer]
Here is the steps to bypass password protection programatically: https://vercel.com/docs/errors#errors/bypassing-password-protection-programmatically

**Reply by rafael-pina** [reply]
Thank you so much 🌟
